adapters/adf/adftest/exemplary/single-banner-pricetype-gross-extend-ext.json

```diff
@@ -0,0 +1,122 @@
+{
+  "mockBidRequest": {
+    "id": "test-request-id",
+    "ext":{
+      "prebid":{
+        "aliases":{
+          "adfalias": "adf"
+        }
+      }
+    },
+    "imp": [{
+      "id": "test-imp-id",
+      "ext": {
+        "bidder": {
+          "mid": "828782",
+          "priceType": "gross"
+        }
```

Comment:
Please add a test case where there are 2 imps in request with different `priceType`

Reply:
Not sure if that is really needed but I can add one.

Reply:
Done. Thank you @VeronikaSolovei9 for the feedback!